esmvaltool/preprocessor/_regrid.py

```diff
@@ -23,8 +21,11 @@
 from iris.analysis import AreaWeighted, Linear, Nearest, UnstructuredNearest
 from numpy import ma
 
+from ..cmor.table import CMOR_TABLES
+
 # Regular expression to parse a "MxN" cell-specification.
-_CELL_SPEC = re.compile(r'''\A
+_CELL_SPEC = re.compile(
+    r'''\A
                             \s*(?P<dx>\d+(\.\d+)?)\s*
```

Comment:
align the string so it is easy to read again: yapf does not change the content of strings